amoro-common/src/main/java/org/apache/amoro/process/ProcessStatus.java

```diff
@@ -26,6 +26,7 @@ public enum ProcessStatus {
   RUNNING,
   SUCCESS,
   CLOSED,
```

Comment:
CLOSED should be removed for I don't see CLOSED being used again.

Reply:
> Check OptimizingQueue.java

i still can't found CLOSED using in OptimizingQueue.java. Please share me the code